### ISSUE TITLE
Add test to read aloud book for language changes

### ISSUE THREAD

**Issue by ways2read** [daisy/epub-accessibility-tests#58]
Additional languages in an ebook should be marked up in the XHTML. Then the correct voice is spoken by a screen reader and the correct translation table is used for braille.

We should add a test to check whether the read-aloud voice switches correctly for read-aloud.

**Comment by GeorgeKerscher** [comment]
A 2023 milestone.

**Comment by marisademeglio** [comment]
Does the experimental read aloud test [AdvancedReadAloud-200](https://github.com/daisy/epub-accessibility-tests/blob/main/content/epub30-test-0370/EPUB/xhtml/advanced_read_aloud_tests.xhtml#L13) close this issue?
